content/en/install.md

```diff
@@ -15,22 +15,11 @@ If you use `conda`, you can install it with:
 conda install numpy
 ```
 
-or with:
-
-```bash
-conda install -c conda-forge numpy
-```
-
 ## pip
```

Comment:
I might list pip above conda, because I figure that most people with conda either already have numpy (installed Anaconda and using the default environment) or know enough to get it anyway. But I'm sure you can make a case the other way too.

Reply:
I was thinking:
- if people have `pip` but not `conda`, they either know to read on, or if they think "what's conda" then it's in their benefit to learn. Too many resources like the Python Packaging User Guide are not explaining conda at all.
- if people have `conda`, they should use it - we've seen a _lot_ of cases of people using `pip install numpy` within conda.